src/main/groovy/org/robolectric/gradle/Configuration.groovy

```diff
@@ -0,0 +1,67 @@
+package org.robolectric.gradle
+
+import org.gradle.api.Project
+import com.android.build.gradle.AppPlugin
+import com.android.build.gradle.LibraryPlugin
+import com.android.build.gradle.api.BaseVariant
+
+class Configuration {
+    private final Project project
+    private final boolean hasAppPlugin
+    private final boolean hasLibPlugin
+    private static final String DEBUG_VARIANT_NAME = "debug"
```

Comment:
Android's Gradle plugin allows running the release variants of tests (I know that Robolectric has historically skipped the release variants).  So this is going to use the debug version of assets, resources, and Manifest, regardless of the tested variant, correct?

I also have a 3rd build type, "staging", which is used for QA to run a release-signed build that still has debugging and logging enabled.

Most of the tests should still run fine, regardless of which variant exposes the resources, so it's not a big deal -- but it should be explained that you won't be able to have a set of release-only or otherBuildType-only tests, because the resources won't be available.

Also, is this going to work with ProductFlavors?  I.e., if I have a flavor for "gingerbread" and one for "full";  I have thus far been running the `:testFullDebug` task.  But will this be able to find the assets and resources if the `DEBUG_VARIANT_NAME` does not incorporate the ProductFlavor?  Technically a Variant is the matrix combinations of BuildType + ProductFlavor.  So if you have any product flavors, there will be no variant that matches the `debug` variant name -- is that correct?